### PR TITLE
Safer raws checks

### PR DESCRIPTION
**Pull request by ai**
> Which issue, if any, is this issue related to?

Manually created AST node without `raws` or custom syntaxes like `postcss-js` or SugarSS.

> Is there anything in the PR that needs further explanation?

No, it's small fix.

### CODE REVIEW

File: src/rules/block-closing-brace-empty-line-before/index.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default function (expectation) {
       if (!hasBlock(statement) || hasEmptyBlock(statement)) { return }
 
       // Get whitespace after ""}", ignoring extra semicolon
-      const before = statement.raws.after.replace(/;+/, "")
+      const before = (statement.raws.after || '').replace(/;+/, "")
```
Comment (ntwb) [inline src/rules/block-closing-brace-empty-line-before/index.js]: Double quotes please, i.e `(statement.raws.after || "")`

File: src/rules/block-closing-brace-newline-before/index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default function (expectation) {
       if (!hasBlock(statement) || hasEmptyBlock(statement)) { return }
 
       // Ignore extra semicolon
-      const after = statement.raws.after.replace(/;+/, "")
+      const after = (statement.raws.after || '').replace(/;+/, "")
```
Comment (ntwb) [inline src/rules/block-closing-brace-newline-before/index.js]: Double quotes please, i.e `(statement.raws.after || "")`

**Comment by ai** [comment]
@ntwb fixed

**Comment by evilebottnawi** [comment]
@ai only `after` or all `raws` can be `undefined` (`null`) ?

**Comment by ai** [comment]
@evilebottnawi `raws` is always a object. But any of its property could be missed.

**Comment by davidtheclark** [comment]
Thanks so much @ai!

**Pull request merged**